src/components/Footer/FooterLinks.js

```diff
@@ -66,7 +66,10 @@ const FooterLinks = () => (
             <Social>
               {social.map(link => (
                 <li key={link.label}>
-                  <ExternalAnchor href={link.to}>
+                  <ExternalAnchor
+                    href={link.to}
+                    title={`yld ${link.label} Account`}
```

Comment:
is there a guidance on the capitalisation of YLD? Is it YLD or yld? Currently the website homepage has YLD, so it may be best to stay consistent with this?

Reply:
every time I go to write YLD/yld/Yld/etc. I think of this slack conversation 

https://yld.slack.com/archives/C04RXJ4UG/p1552906659073800

Reply:
I believe `yld` is the proper way